Dockerfile

```
@@ -1,14 +1,32 @@
+
+RUN curl -L https://github.com/acoustid/chromaprint/archive/41a3e8fb3eb907d7a0338ada291982672a2226df.zip -o 41a3e8fb3eb907d7a0338ada291982672a2226df.zip && \
+    unzip 41a3e8fb3eb907d7a0338ada291982672a2226df.zip && \
+    cd chromaprint-41a3e8fb3eb907d7a0338ada291982672a2226df && \
+    mkdir build && cd build && \
+    cmake .. -DCMAKE_INSTALL_PREFIX=/usr -DCMAKE_C_FLAGS="-O3 -march=haswell -ffast-math" -DCMAKE_CXX_FLAGS="-O3 -march=haswell -ffast-math" -DCMAKE_BUILD_TYPE=Release && \
+    make -j$(nproc) && \
+    make install DESTDIR=/opt/chromaprint/install
 
 FROM ubuntu:24.04 AS base
 
 RUN apt-get update && \
```

Hadolint (2.12.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)